File: backend/src/main/java/com/corenetworks/ProyectoFinal/modelo/Publicaciones.java
package com.corenetworks.ProyectoFinal.modelo;

import java.time.LocalDate;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Entity
@Table(name = "publicaciones")
public class Publicaciones {
    private String descripcion;
    private int cantidadLikes;
    private String contenido;
    private String urlCompartir;
    //private String guardar;
    private LocalDate fCreacion;

}

File: frontend/ig_clon/src/app/_modelo/Publicaciones.ts
import { Comentario } from "./Comentario";
import { Usuario } from "./Usuario";

export interface Publicaciones{
    idPublicacion:number,
    descripcion:string,
    cantidadLikes:number,
    rutaContenido:string,
    urlCompartir:string,
    fCreacion:Date,
    hCreacion:Date,
    usuario:Usuario,
    comentario:Comentario[]
}